.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+    "*.(ts|tsx)": ["eslint --fix", "prettier --write"]
```

Comment:
I'm not sure this will work quite right as we use a different eslintrc file depending on if the file is in `src`, `test` or `scripts`